inetum-realjobs-backend/src/main/resources/application.properties

```diff
@@ -21,6 +21,8 @@ application.mail.from=noreply@realjobs.com
 spring.jpa.properties.javax.persistence.schema-generation.scripts.action=create
 spring.jpa.properties.javax.persistence.schema-generation.scripts.create-target=create-tables.sql
 spring.jpa.properties.javax.persistence.schema-generation.scripts.create-source=metadata
+spring.jpa.properties.hibernate.hbm2ddl.schema-generation.script.append=false
+spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
```

Comment:
What's this one for?

Reply:
I had something fail due to an empty bean error. Might not be needed anymore.